src/pages/saksbehandling/sakintro/Sakintro.tsx

```diff
@@ -109,7 +109,7 @@ const Sakintro = (props: { sak: Sak }) => {
                     <AvslåtteSøknader sak={props.sak} avslåtteSøknader={avslåtteSøknader} intl={intl} />
                     <LukkedeSøknader lukkedeSøknader={lukkedeSøknader} intl={intl} />
                     <AvsluttedeRevurderinger avsluttedeRevurderinger={avsluttedeRevurderinger} intl={intl} />
-                    {klageToggle && <KlageLister sakId={props.sak.id} klager={props.sak.klager} />}
+                    {klageToggle && <KlageLister sak={props.sak} klager={props.sak.klager} />}
```

Comment:
Ettersom det er så få props, så synes jeg det er bedre å sende inn det komponenten faktisk trenger. Dvs fortsette å sende inn `sakId`, og dermed lage et nytt prop for `vedtak`. Komponenter som tar inn spesifikke props er generelt mye enklere å gjenbruke.

Hvis ikke, så ville jeg i så fall sløyfet `klager`-propet, siden det nå blir sendt inn to ganger (ligger på sak som blir sendt inn i sin helhet)

Reply:
fikser